modules/core/shared/src/main/scala/lucuma/core/enum/package.scala

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core
+
+import java.time.Month
+import java.time.Month._
+import cats.implicits._
```

Comment:
Will this get the cats updates, (e.g. to `import cats.syntax.all._`)?

Reply:
Yup, slipped through the cracks. Sending another commit.